internal/pkg/utils/reflecthelper/reflecthelper.go

```diff
@@ -54,7 +55,7 @@ func StringFromOneTaggedField(tag string, model interface{}) (str string, found
 		return "", false
 	}
 
-	reflection := reflect.ValueOf(model).Elem()
+	reflection := unwrap(reflect.ValueOf(model))
```

Comment:
`Elem()` ti vrati vnorenu hodnotu ak sa jedna o pointer alebo interface - tj. bez tejto zmeny to nefungovalo s hodnotou (sktrukturou), ktora nebola "zabalena" do pointra alebo interface.